platform/src/test/java/platformfeatures/PF_Withdrawal.feature

```diff
@@ -0,0 +1,60 @@
+
+#Author: Sahaj Balgunde
+#Keywords Summary : Withdawal Test with Checkbox and Without Checkbox
+
+
+
+Feature: Withdraw user account
+@FunctionalTest 
+Scenario Outline: User is withdrawing the account
+
+#Given User is on Withdrawal Page
+#When User clicks on accept checkbox
+#Then the check box is clickable
+
+
+#Scenario: User is withdrawing the account
+
+Given Open Browser and URL <browser>
+Then User is on Withdrawal Page ticks the checkbox and withdraw account
```

Comment:
Include user actions in When statement
Validate test in Then statement

Reply:
Updated as requested.